assets/sass/components/wp-dashboard/_googlesitekit-wp-dashboard.scss

```
@@ -64,6 +64,69 @@
 		}
 	}
 
+	.googlesitekit-wp-dashboard-loading {
+
+		.googlesitekit-wp-dashboard-stats {
+			display: block;
+		}
+
+		.googlesitekit-preview-block {
+			&.googlesitekit-wp-dashboard-loading {
+				min-height: 30px;
+			}
+
+			&.googlesitekit-wp-dashboard__cta {
+				min-height: 18px;
+			}
+
+			&.googlesitekit-wp-dashboard-loading__search_console_active_and_connected,
+			&.googlesitekit-wp-dashboard-loading__can_view_analytics {
+				min-height: 95px;
+			}
+
+			&.googlesitekit-wp-dashboard-stats__cta {
+				min-height: 300px;
+				width: 100%;
+			}
+
+			&.googlesitekit-unique-visitors-chart-widget {
+				min-height: 320px;
+			}
+
+			&.googlesitekit-search-console-widget {
+				min-height: 580px;
+
+				@media (max-width: $bp-mobileOnly) {
+					min-height: 330px;
+				}
+			}
+		}
+	}
+
+	.googlesitekit-wp-dashboard-analytics-activate-cta {
+		min-height: 310px;
+	}
+
+	.googlesitekit-wp-dashboard-analytics_active_and_connected {
+		min-height: 935px;
+
+		@media (max-width: $bp-mobileOnly) {
+			min-height: 780px;
+		}
+
+		.googlesitekit-wp-dashboard-search_console_active_and_connected {
```

You missed the & here which will ensure the classes are concatenated property without a space:

Here's the currently generated selector:

```css
#google_dashboard_widget .googlesitekit-wp-dashboard-analytics_active_and_connected .googlesitekit-wp-dashboard-search_console_active_and_connected
```

We need to prefix this class with an `&` to get the following result, retaining the selector as it was before the recent commit:

```css
#google_dashboard_widget .googlesitekit-wp-dashboard-analytics_active_and_connected.googlesitekit-wp-dashboard-search_console_active_and_connected 
```

```suggestion
		&.googlesitekit-wp-dashboard-search_console_active_and_connected {
```